src/ResourceManager/HDInsight/Commands.HDInsight/help/Add-AzureRmHDInsightConfigValues.md

```diff
@@ -232,6 +234,58 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Spark2Defaults
+Specifies the Spark2 Defaults configurations of this HDInsight cluster.```yaml
```

Comment:
@jeffersonezra can you fix this file so that the `` ```yaml `` is on a line by itself. For example:

````
### -RServer
Specifies the RServer configurations. Valid only for RServer clusters.
```yaml
Text
```
````